test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4297,7 +4297,8 @@ def _load_configuration(self, infile, objtype=None):
                                               + "configuration: %s" % k)
                             return False
                         with open(fn, 'w') as fd:
-                            fd.write("\n".join(v))
+                            mom_config_data = "\n".join(v) + "\n"
```

Comment:
I am confused in the fix, PR title and fix says it is for the mom config file, but if I am not wrong, this place is for sched object (see line 4290, `elif objtype == MGR_OBJ_SCHED`)

Reply:
@hirenvadalia Thanks for catching it. I have done the changes.